dcpy/utils/data.py

```diff
@@ -164,7 +161,13 @@ def unzip_file(zipped_filename: Path, output_dir: Path) -> list[str]:
 
     with zipfile.ZipFile(zipped_filename, "r") as zip_ref:
         zip_ref.extractall(output_dir)
-        extracted_files = zip_ref.namelist()
+        extracted_files = set()
+
+        for info in zip_ref.infolist():
+            extracted_files.add(info.filename.rstrip("/"))
+            dir = os.path.dirname(info.filename).rstrip("/")
+            if dir:
+                extracted_files.add(dir)
```

Comment:
was this needed for `unzipped_filename: lion/lion.gdb` in the next commit to work?